conf/airframes/AGGIEAIR/aggieair_rp3_lia.xml

```diff
@@ -35,6 +35,8 @@ RP3 Lisa MX
 
     <define name="AGR_CLIMB" />
     <define name="POLY_OSAM_HALF_SWEEP_ENABLED" value="FALSE"/>
+    <define name="AIRSPACE"/>
+    <define name="DATALINK_LOST_TIME" value="45"/>
```

Comment:
I am thinking about the HOME_MODE_DATALINK_LOST_TIME  as a safety feature you want to bound to the airframe, rather than a particular flight plan. In which case you can still define an exception in the flight plan (with different values for different flights lets say), but if you define HOME_MODE_DATALINK_LOST_TIME in the airfame config file then you will have sort two independent checks. So I don't think it makese sense to define this in the flight plan (since you can use an exception to do the same, and this was kind of a way to avoid using exceptions in the first place). Does that make sense?